data-visualization-of-the-week-template.ipynb

```diff
@@ -22,6 +22,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "### Instructions:\n",
+    "#### Step 1 (your only step): “Run” the cells to see the graphs\n",
+    "Click “Cell” and select “Run All.” This will import the data and run all the code to make this weeks data visualization (scroll down after you’ve run the cells).  **You don’t need to do any coding if you don't want to**.\n",
+    "\n",
+    "![instructions](resources/instructions.png)"
```

Comment:
change folder name to "images", so change path to:

images/instructions.png

Reply:
Directory name changed and ipynb adjusted.